manifests/server.pp

```diff
@@ -180,6 +182,19 @@
       ensure  => $api_package_version
     }
 
+    # wazuh-api config.js
+    # this hash is currently only covering the basic config section of config.js
+    # TODO: allow customization of the entire config.js
+    # for reference: https://documentation.wazuh.com/current/user-manual/api/configuration.html
+    file { '/var/ossec/api/configuration/config.js':
+      content => template($api_config_template),
+      owner   => 'root',
+      group   => 'root',
```

Comment:
Hi @rafaelfc-olx 

The right group is ossec

JN

Reply:
just changed, thanks again for the heads up :)